### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by pjeweb**
## Describe your changes

Explain generating secrets better and add python option. Also improve various other things in the README

## Notes for testing your change

Read the README and try the new suggested commands.

### CODE REVIEW

**Comment by MattIPv4** [comment]
Instead of doing `"Generate secrets"`, we could do `[Generate secrets](#generate-secrets)` to deep-link. Or, you could get really fancy and do `[Generate secrets]` and then `[Generate secrets]: #generate-secrets` at the end of the file I think?

File: README.md
Comment (pjeweb) [inline README.md:94]: No need to specify Linux, macOS, WSL here, you can install openssl.exe on windows as well.

File: README.md
Comment (pjeweb) [inline README.md:79]:
```suggestion
   5. The data encryption passphrase (`DATA_ENCRYPTION_PASSPHRASE`) has to be filled with a 24-byte Base64-encoded secret. See "Generate secrets" below.
```

**Comment by MattIPv4** [comment]
@pjeweb do you want me to take this branch over and get it landed?

**Comment by pjeweb** [comment]
> @pjeweb do you want me to take this branch over and get it landed?

yes please. sorry i forgot about this completely :)

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Comment by MattIPv4** [comment]
@pjeweb I'll let you merge this if my commit looks good, so it is "reviewed" by you given it was my push

**Pull request merged**
